tasks/promote.yml

```diff
@@ -4,6 +4,28 @@
     msg: "os_images_name_suffix is empty please provide it."
   when: os_images_name_suffix is defined and os_images_name_suffix | length == 0
 
+- name: Ensure images for retirement exist
+  os_image:
```

Comment:
wouldn't os_image_info be better here? I understand os_image will try to upload the image if it does not exists but no filename given, probably better would be to run os_image_info and use assert module that the image exists.

Reply:
os_image_info has no error handling, so that would have to be added on top of information parse.
I have added another information l task with explanation of what the ephemeral ephemeral os_image error means. Hopefully that's ok.

Reply:
It's done